src/lib/es2015.symbol.wellknown.d.ts

```diff
@@ -150,7 +150,7 @@ interface Promise<T> {
 }
 
 interface PromiseConstructor {
-    readonly [Symbol.species]: Function;
+    readonly [Symbol.species]: PromiseConstructor;

```

Comment:
[Per the spec](https://tc39.github.io/ecma262/#sec-get-promise-@@species), the property type should be `this`.

Reply:
@DanielRosenwasser, do you know if using a `this` type here would be problematic for someone writing a subclass?